README.md

```diff
@@ -4,6 +4,8 @@
 ![Unit Tests](https://github.com/GoogleChromeLabs/chromium-bidi/actions/workflows/unit.js.yml/badge.svg)
 ![WPT Tests](https://github.com/GoogleChromeLabs/chromium-bidi/actions/workflows/wpt.yml/badge.svg)
 
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit)](https://github.com/pre-commit/pre-commit)
```

Comment:
if it does not run locally, why is it called pre-commit?

Reply:
It does run locally, as well.

You can do `brew install pre-commit` + `pre-commit install`, then it will always run locally upon `git commit`, `git merge`, `git rebase`, etc.

The CI is an extra layer on top e.g. to prevent people who forgot to add local pre-commit integration.